src/components/Marketplace/FarmCard.js

```diff
@@ -1,12 +1,12 @@
 import React, { useState } from 'react';
-import './FarmCard.css';
 import {
   Button, Card, CardContent, Chip, Collapse, Grid, IconButton, Typography,
 } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import ExpandLessIcon from '@material-ui/icons/ExpandLess';
 import PropTypes from 'prop-types';
+import './Font.css';
```

Comment:
This is a great change. So what we're planning is creating a folder for our font and we can all pull that styling from there.

So inside of our assets folder, create a styling folder, and make a css file called font.css. Then put the font import there